### ISSUE TITLE
After playing for a few minutes.......the whole world starts to tilt at an angle....VIVE

### ISSUE THREAD

**Issue by SGTPinchaloaf** [KozGit/DOOM-3-BFG-VR#237]
I thought this was my Tracking getting messed up but all my other games are working fine. Somehow the world starts to gradually display crooked.......Like if you were to mess with one of your lighthouses and the ground started to tilt at an angle and you have to run the room scale setup to fix it. Its just in this game though?? has anybody else experienced this???

**Comment by KozGit** [comment]
The word tilting is not an issue I've encountered before - if anyone else has experienced this please let me know.  Offhand I can't think of any reason this would be happening - tilt is only applied in one place just prior to rendering. The value isn't stored so there's no real way a cumulative error should creep in, the full tilt value is read from the headset each frame and applied.  

The hud not turning with you is an issue if you have it locked to your body in the options.  ( In v .021 the body does not turn when you turn in real life - this is fixed in the upcoming version ). In the options you can change the hud to be locked to your view instead of the body.  In the menus go to Settings->Vr Options->UI Options->HUD options and change 'Location' to 'Lock to view'. ( moving forward with analog controls while playing also makes the body/hud face forward )

**Comment by SGTPinchaloaf** [comment]
I got the hud fixed....thankyou . Ill mess with the settings and see if its just some weird anamoly in my setup. 

Any idea when you guys will release another update.....sorry if youve been asked this 100 times

**Comment by SGTPinchaloaf** [comment]
Its still doing it even after trying to change a bunch of different settings ........but this is what i noticed:

I keep the chaperone floor border on all the time so i can see it as it happens. When I look left or right I can see the world slightly tilting separate from my chaperone border .....if that makes any sense. However if i look up or down the chaperone border does not move. Just looking left and right. Under constant locomotion and constant looking around it tends to get worse the more I look around and it is very nausea inducing. Sometimes it will correct but other times it gets worse. 

My brother also has a vive and plays this game. he has also noticed this in his game too.

**Comment by Slinkywinky** [comment]
Do you have a controller or joystick also plugged into your computer. I had an issue similar to what your experiencing in another game and it turned out to be my joystick. I just adjusted the dead zone and it fixed the problem or you can just unplug it.

**Comment by SGTPinchaloaf** [comment]
No....no joystick or any other controller plugged in and the mouse is a cordless laser mouse.....i know it isnt moving. The only thing i see out of place is the chaperone border movement when i look left or right......no other game i have does this
